### DISCUSSION TITLE
SEB Windows Client 3.3.1 Failed to Connect with Jitsi Video Conference Service

### DISCUSSION THREAD

**Discussion by ano010** [SafeExamBrowser/seb-win-refactoring#281] [Q&A]
I have been trying to set up an exam proctoring system using the SEB server, Moodle, and Jitsi since last month. The setup is almost done. The integrated system works with the SEB iOS client 3.0.1 but the SEB client does not get connected to the Jitsi video conference service with SEB Window client 3.3.1. It shows "https's server IP address could not be found" in the Jitsi frame on the SEB client side. Can I get any clue on the issues with the Windows SEB client?

I hereby attached the screenshot of the SEB client issue.

![processed-e1d9772b-be1c-4838-aa78-32f8aeb2f181_cp2sF7UT](https://user-images.githubusercontent.com/33832254/145346379-d416a7f4-3573-4263-85d3-2a234a863dff.jpeg)

**Comment by dbuechel** [comment]
The proctoring implementation with Jitsi is still experimental. However, if I remember correctly, there was some weird issue where the URL for the server would need to be specified without "https://".

**Reply by ano010** [reply]
Thanks for the reply. But if I set the URL without "https://", there is a validation error in the UI.

<img width="578" alt="image" src="https://user-images.githubusercontent.com/33832254/146311799-57f310db-0cfe-48e7-b1a7-003c0cbfe5c9.png">

**Reply by ano010** [reply]
I checked to change the URL in the database without "https://" but it throws the same error that the URL pattern is wrong.
